elide-core/src/main/java/com/yahoo/elide/Elide.java

```diff
@@ -184,9 +188,13 @@ public ElideResponse get(String path, MultivaluedMap<String, String> queryParams
             JsonApiDocument jsonApiDoc = new JsonApiDocument();
             RequestScope requestScope = new RequestScope(path, apiVersion, jsonApiDoc,
                     tx, user, queryParams, requestId, elideSettings);
-            requestScope.setEntityProjection(new EntityProjectionMaker(elideSettings.getDictionary(),
-                    requestScope).parsePath(path));
+            EntityProjection projection = new EntityProjectionMaker(elideSettings.getDictionary(),
+                    requestScope).parsePath(path);
+            requestScope.setEntityProjection(projection);
             BaseVisitor visitor = new GetVisitor(requestScope);
+
+            elideAcceptQuery(requestScope, opaqueUser, requestScope.getQueryParams(), path);
```

Comment:
THe order here is wrong.  we have to accept the query before we build the entity projection.  THe entity projection includes parsing code that can throw an exception/error.  Those queries wont' get logged.

Also - accepting a query could move to the handleRequest function

Reply:
Fixed in most recent commit